### PR TITLE
Separate lambdas for each chunked-worker client.

### PR DESCRIPTION
**Pull request by ttung**
Automatically builds makefile rules from the client list, and rewrites the config for each separate client.

Connects to #319 
Connects to #393

### CODE REVIEW

File: daemons/dss-chunked-task/clients.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+import os
+import sys
+
+pkg_root = os.path.abspath(os.path.join(os.path.dirname(__file__), "..", ".."))  # noqa
+sys.path.insert(0, pkg_root)  # noqa
+
+from dss.events.chunkedtask import aws
+
+for name in aws.get_clients().keys():
+    print(f"dss-chunked-task-{name}")
```
Comment (kislyuk) [inline daemons/dss-chunked-task/clients.py]: Consider moving this entry point into /scripts/.

File: iam/policy-templates/dss-chunked-task-s3_copy_write_metadata-lambda.json [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    {
+      "Effect": "Allow",
+      "Action": "sns:Publish",
+      "Resource": [
+        "arn:aws:sns:*:$account_id:dss-*-$stage"
+      ]
+    },
+    {
+      "Effect": "Allow",
+      "Action": "s3:*",
```
Comment (kislyuk) [inline iam/policy-templates/dss-chunked-task-s3_copy_write_metadata-lambda.json]: Consider enumerating permissions on a need basis here (to be fair, we may need to do it in other policies too). For example, to limit exposure to catastrophic data loss, we may want to deny delete access to all daemons except a specially managed deleter daemon.

File: daemons/dss-chunked-task/clients.py (hunk 1 above)
Comment (ttung) [reply daemons/dss-chunked-task/clients.py]: fixed.

File: iam/policy-templates/dss-chunked-task-s3_copy_write_metadata-lambda.json (hunk 2 above)
Comment (ttung) [reply iam/policy-templates/dss-chunked-task-s3_copy_write_metadata-lambda.json]: done.

**Pull request merged**